src/usr/local/buildpack/tools/gradle.sh

```diff
@@ -68,12 +68,12 @@ if [[ -z "${tool_path}" ]]; then
 
   URL="https://services.gradle.org/distributions"
 
-  curl -sSfLo ${file} ${URL}/gradle-${TOOL_VERSION}-bin.zip
-  unzip -q -d ${base_path} ${file}
+  curl -sSfLo ${file} ${URL}/gradle-"${TOOL_VERSION}"-bin.zip
+  unzip -q -d "${base_path}" ${file}
   rm ${file}
-  mv ${base_path}/${TOOL_NAME}-${TOOL_VERSION} ${tool_path}
+  mv "${base_path}"/"${TOOL_NAME}"-"${TOOL_VERSION}" "${tool_path}"
```

Comment:
Could be simplified, but is OK as well.

Reply:
yes, the autofix is pretty plain. Feel free to make suggestions here to commit. 🙃

Reply:
```suggestion
  mv "${base_path}/${TOOL_NAME}-${TOOL_VERSION}" "${tool_path}"
```